isic_archive/tasks/notification.py

```diff
@@ -12,13 +12,14 @@ def maybeSendIngestionNotifications():
         if not Batch().hasImagesPendingIngest(batch):
             failedImages = list(Batch().imagesFailedIngest(batch))
             skippedFilenames = list(Batch().imagesSkippedIngest(batch))
-            sendIngestionNotification.delay(batch['_id'], failedImages, skippedFilenames)
+            numImages = Batch().images(batch).count()
```

Comment:
FYI, MongoDB has [deprecated `Cursor.count`](https://api.mongodb.com/python/current/api/pymongo/cursor.html#pymongo.cursor.Cursor.count). However, I don't see a sane alternative, given the current idioms of the model layer.